rllib/offline/offline_data.py

```diff
@@ -146,8 +147,7 @@ def sample(
                 # Add constructor `kwargs` when using remote learners.
                 fn_constructor_kwargs.update(
                     {
-                        "learner": self.learner_handles,
-                        "locality_hints": self.locality_hints,
+                        "learner": None,
```

Comment:
Wait, what changed here?

Reply:
Locality hints are gone. The learner will go in a third PR blocked by the Ray Data w/ Ray Tune issue